source/blog/2017-02-16-hyperconverged-ovirt-with-ovn-part1.md

```diff
@@ -0,0 +1,203 @@
+* OVN, **Open Virtual Network**, is part of the [Open vSwitch (OVS)](http://openvswitch.org/) project: it complements the existing capabilities of OVS to add native support for virtual network abstractions, such as virtual L2 and L3 overlays and security groups. Since release 4.1, oVirt ships an OVN provider to be used to consume and manage OVN networks from oVirt.
+* [Cockpit](http://www.cockpit-project.org/) is an **interactive web-based server admin interface**.
+
+In this tutorial we'll see how to deploy a test environment deploying oVirt hosted-engine in an hyper-converged way consuming OVN networks; the deployment will be mainly performed in a graphical way using Cockpit.
+
+## Prerequisites
+
+* Three hosts (or three VMs enabling the nested virtualization support on your physical host) called also nodes; each node should have at least 6 GB of RAM, two disks (20GB for local storage and 200GB for GlusterFS) and single nic (three nics with 10GbE for the storage network are strongly advised for production systems).
+* Each node should be installed with a fully updated CentOS-7 (1611).
+* For simplicity we avoid requiring DNS and DHCP infrastructure although they are strongly recommended for production deployments. In this tutorial we'll simply use static IPv4 addressing and /etc/hosts manually synchronized on all the hosts to resolve the hostnames. The naming schema we are going to use is:
+```
+192.168.1.200 enginevm.localdomain enginevm
+192.168.1.201 demovm1.localdomain demovm1
+192.168.1.202 demovm2.localdomain demovm2
+192.168.1.203 demovm3.localdomain demovm3
+```
+'enginevm.localdomain' is the hostname we are going to use for the virtual machine we are going to create for the engine; 'demovm{1,2,3}.localdomain' are the hostnames we are going to use on our three hosts.
+
+## Initial steps
+
+Fill /etc/hosts on all the three hosts:
+```
+# Add hosts to /etc/hosts
+cat <<EOF >> /etc/hosts
+192.168.1.200 enginevm.localdomain enginevm
+192.168.1.201 demovm1.localdomain demovm1
+192.168.1.202 demovm2.localdomain demovm2
+192.168.1.203 demovm3.localdomain demovm3
+EOF
+```
+On all the three hosts, add oVirt and gdeploy repositories (gdeploy packages are not included in oVirt 4.1 release but they will be in the next minor release):
+```
+yum install -y http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+
+#Add gdeploy repo
+cat <<EOF > /etc/yum.repos.d/gdeploy.repo
+[rnachimu-gdeploy]
+name=Copr repo for gdeploy owned by rnachimu
+baseurl=https://copr-be.cloud.fedoraproject.org/results/rnachimu/gdeploy/epel-7-x86_64/
+type=rpm-md
+skip_if_unavailable=True
+gpgcheck=1
+gpgkey=https://copr-be.cloud.fedoraproject.org/results/rnachimu/gdeploy/pubkey.gpg
+repo_gpgcheck=0
+enabled=1
+enabled_metadata=1
+EOF
+```
```

Comment:
Repo can be added by 'yum-config-manager --add https://copr.fedorainfracloud.org/coprs/rnachimu/gdeploy/repo/epel-7/rnachimu-gdeploy-epel-7.repo.